### PR TITLE
Wait for Jobs

### PR DESCRIPTION
**Pull request by mount986**
New parameter to set the --wait-for-jobs flag:
`--wait-for-jobs                if set and --wait enabled, will wait until all Jobs have been completed before marking the release as successful. It will wait for as long as --timeout`

### CODE REVIEW

File: assets/out [hunk 1]
```diff
@@ -148,8 +149,13 @@ helm_upgrade() {
...
   fi
 
-  if [ "$check_is_ready" == "true" ]; then
+  if [ "$wait_for_jobs" == "true" ]; then
     upgrade_args+=("--wait")
+    upgrade_args+=("--wait-for-jobs")
+  else
+    if [ "$check_is_ready" == "true" ]; then
+      upgrade_args+=("--wait")
+    fi
```
Comment (Typositoire) [inline assets/out:158]: Please replace with `elif`

**Review by Typositoire — CHANGES_REQUESTED** [review]

**Comment by Typositoire** [comment]
Also bump the minor version in `VERSION` file

**Comment by Typositoire** [comment]
Concourse: Version bump did not happen. v1.25.0 is the same as v1.25.0

File: assets/out (hunk 1 above)
Comment (mount986) [reply assets/out]: fair enough

**Comment by Typositoire** [comment]
Concourse: Version bump did not happen. v1.25.0 is the same as v1.25.0

**Comment by Typositoire** [comment]
Concourse: Version bump found.

**Comment by Typositoire** [comment]
Concourse: Version bump found.

**Comment by Typositoire** [comment]
Concourse: Version bump found.

**Review by Typositoire — APPROVED** [review]

**Pull request merged**